File: lab3.1.cpp
#include <iostream>
using namespace std;
int main() {
    int x=0;
    cout<<"Enter a number between 10 and 100 ";
    cin>>x;
    while(x < 10 or x > 100) {
        cout<<"Enter a number between 10 and 100 ";
        cin>>x;
    }
    int fizz=0;
    int buzz=0;
    int fizzbuzz=0;

    for(int i=1;i<=x;i++) {
        if (i%7==0) {
            continue;
        }
        if (i%3==0 && i%5==0) {
            cout<<"Fizzbuzz"<<endl;
            fizzbuzz++;
        }
        if (i%3==0) {
            cout<<"fizz"<<endl;
            fizz++;
        }
        if (i%5==0) {
            cout<<"buzz"<<endl;
            buzz++;
        }
        else {
            cout<<i<<endl;
        }
        cout << "Summary:\n" << endl;
        cout << "Fizz count: " << fizz << endl;
        cout << "Buzz count: " << buzz << endl;
        cout << "FizzBuzz count: " << fizzbuzz << endl;
    }

    return 0;
}

File: LAB2/lab3.1.py
x= int(input("enter the number between 10 and 100"))
while x < 10 or x > 100:
    x = int(input("enter the number between 10 and 100"))

fizz=0
buzz=0
fizzbuzz=0
for i in range(1,x+1):
    if i % 7 == 0:
        continue
    if i%3==0 and i%5==0:
        print("fizzbuzz")
        fizzbuzz=fizzbuzz+1
    elif i%3==0:
            print("fizz")
            fizz=fizz+1
    elif i%5==0:
            print("buzz")
            buzz=buzz+1
    else:
            print(i)
print("Finally\n")
print("Fizz count ",fizz )
print("Buzz count ",buzz)
print("Fizzbuzz count ",fizzbuzz)




